packages/esm-patient-search-app/src/ui-components/empty-data-illustration.component.tsx

```diff
@@ -1,40 +1,40 @@
 import * as React from 'react';
 
-const EmptyDataIllustration = ({ width = '74', height = '72' }) => {
+const EmptyDataIllustration = ({ width = '72', height = '72' }) => {
```

Comment:
Do you have a screen shot of this? Asking because I see the viewBox is still 74x72, although I'm guess those pixels are unimportant.

Reply:
Yep. Inconsequential.

![CleanShot 2023-10-27 at 12  58 03@2x](https://github.com/openmrs/openmrs-esm-patient-management/assets/8509731/b0e1d733-9644-4a3a-b619-2bbd1647f719)